providers/github/go.mod

```diff
@@ -9,6 +9,7 @@ toolchain go1.22.0
 require (
 	github.com/cockroachdb/errors v1.11.1
 	github.com/google/go-github/v57 v57.0.0
```

Comment:
nit: while you're in here you might want to switch over to v59. These never get updated automatically due to how they version them.

Reply:
Thanks!